README.md

```diff
@@ -218,8 +218,21 @@ Running `amppkg` with the `-invalidcert` flag will skip the check for
 `-development` flag.
 
 Chrome can be configured to allow these invalid certificates with the
-*Allow Signed HTTP Exchange certificates without extension* experiment:
-chrome://flags/#allow-sxg-certs-without-extension
+`--ignore-certificate-errors-spki-list` command line flag:
+
+```
+google-chrome --ignore-certificate-errors-spki-list=<hashes> --user-data-dir=<dir>
+```
+
+where `<hashes>` is a comma separated list of Base64-encoded SHA-256 SPKI
+Fingerprints and it is necessary to specify `--user-data-dir` with a valid
```

Comment:
Maybe "valid or createable directory" or something? I do `--user-data-dir=/tmp/newdir` and Chrome creates it, as long as the parent dir exists.

Reply:
Should be resolved in latest commit.